examples/tfengine/generated/org_foundation/cicd/main.tf

```diff
@@ -64,6 +64,7 @@ locals {
     "roles/logging.configWriter",
     "roles/resourcemanager.projectCreator",
     "roles/resourcemanager.organizationAdmin",
+    "roles/storage.objectAdmin",
```

Comment:
Can we add this role to the logging bucket only instead of doing it at the folder/org level?

Reply:
Done.